specification/alertsmanagement/resource-manager/Microsoft.AlertsManagement/preview/2018-11-02-privatepreview/AlertsManagement.json

```diff
@@ -0,0 +1,2062 @@
+          }
+        },
+        "schedule": {
+          "$ref": "#/definitions/suppressionSchedule"
+        }
+      }
+    },
+    "suppressionSchedule": {
+      "type": "object",
+      "description": "Schedule for a given suppression configuration.",
+      "properties": {
+        "startDate": {
+          "type": "string",
+          "description": "Start date for suppression"
+        },
+        "endDate": {
+          "type": "string",
+          "description": "End date for suppression"
+        },
+        "startTime": {
+          "type": "string",
+          "description": "Start time for suppression"
+        },
+        "endTime": {
+          "type": "string",
+          "description": "End date for suppression"
+        },
+        "recurrenceValues": {
+          "type": "array",
+          "description": "Specifies the values for recurrence pattern",
+          "items": {
+            "type": "integer"
+          }
+        }
+      }
+    },
+    "actionRule": {
+      "description": "Action rule object containing target scope, conditions and suppression logic",
+      "allOf": [
+        {
+          "$ref": "#/definitions/Resource"
+        }
+      ],
+      "properties": {
+        "tags": {
+          "type": "object",
+          "description": "List of tags specified by user associated with this resource"
+        },
+        "properties": {
+          "$ref": "#/definitions/actionRuleProperties"
+        }
+      }
+    },
+    "actionRuleProperties": {
+      "description": "Action rule properties defining scope, conditions, suppression logic for action rule",
+      "properties": {
+        "scope": {
+          "$ref": "#/definitions/scope"
+        },
+        "conditions": {
+          "$ref": "#/definitions/conditions"
+        },
+        "suppressionConfig": {
+          "$ref": "#/definitions/suppressionConfig"
+        },
+        "description": {
+          "type": "string",
+          "description": "Description of action rule"
+        },
+        "resourceGroup": {
+          "type": "string",
+          "description": "Resource group where action rule is stored",
+          "readOnly": true
+        },
+        "createdDateTime": {
+          "type": "string",
+          "format": "date-time",
+          "description": "Creation time of action rule. Date-Time in ISO-8601 format.",
+          "readOnly": true
+        },
+        "lastModifiedDateTime": {
+          "type": "string",
+          "format": "date-time",
+          "description": "Last updated time of action rule. Date-Time in ISO-8601 format.",
+          "readOnly": true
+        },
+        "createdUserName": {
+          "type": "string",
+          "description": "Created by user name.",
+          "readOnly": true
+        },
+        "lastModifiedUserName": {
+          "type": "string",
+          "description": "Last modified by user name.",
+          "readOnly": true
+        },
+        "enabled": {
```

Comment:
string enums generally work better longer term. Specially when you want to expand states the property can take.

Reply:
Resolved in latest commit